web/src/Components/Timeline/timeline.tsx

```diff
@@ -9,6 +9,8 @@ interface Props {
   actualTimeline: any;
   loading: boolean;
   method: 'create' | 'view' | 'update';
+  mtgStartYear: any;
```

Comment:
mtgStartYear and mtgRange can be number type

Reply:
Fixed